lupa/__init__.py

```diff
@@ -63,8 +63,10 @@ def _import_newest_lib():
         raise RuntimeError("Failed to import Lupa binary module.")
     # prefer Lua over LuaJIT and high versions over low versions.
     module_name = max(modules, key=lambda m: (m[1] == 'lua', tuple(map(int, m[2] or '0'))))
-
-    _newest_lib = __import__(module_name[0], level=1, fromlist="*", globals=globals())
+    try:
+        _newest_lib = __import__(module_name[0], level=1, fromlist="*", globals=globals())
+    except ModuleNotFoundError:
+        _newest_lib = __import__(module_name[1], level=1, fromlist="*", globals=globals())
```

Comment:
This seems unrelated. Why did you change it?

Reply:
Actually this is related to another bug. If you link to your custom lua libs, the original will fail to find corret module.

Reply:
Ok, I will put it in a seperate pr.